### PR TITLE
Add Camping World spider

### PR DESCRIPTION
**Pull request by reidab**
This adds a spider for Camping World locations via https://rv.campingworld.com/locations

### CODE REVIEW

File: locations/spiders/campingworld.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    def start_requests(self):
+        yield Request(url=self.locator_url, callback=self.parse_locator)
+
+    def parse_locator(self, response):
+        # The locator HTML contains a JWT in a meta tag that we need to make the API call.
+        meta = response.xpath('//meta/@content')
+        meta_content = [s.get() for s in meta]
+        tokens = [s for s in meta_content if s.startswith("eyJ") and ".eyJ" in s]
+        token = tokens[0]
```
Comment (Cj-Malone) [inline locations/spiders/campingworld.py:41]: Would it be better to do this in the xpath?
```suggestion
        token = response.xpath('//meta[starts-with(@content, "eyJ")][contains(@content, ".eyJ")]/@content').get()
```

File: locations/spiders/campingworld.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+from scrapy.http import JsonRequest, Request
+
+import base64
+
+from locations.items import Feature
+from locations.user_agents import BROWSER_DEFAULT
+
+
+class CampingWorldSpider(scrapy.Spider):
+    name = "campingworld"
```
Comment (Cj-Malone) [inline locations/spiders/campingworld.py:13]: If this is US only, we'd prefer to flag that:
```suggestion
class CampingWorldUSSpider(scrapy.Spider):
    name = "camping_world_us"
```

And a filename of `camping_world_us.py`

File: locations/spiders/campingworld.py (hunk 1 above)
Comment (reidab) [reply locations/spiders/campingworld.py]: Works for me.

File: locations/spiders/campingworld.py (hunk 2 above)
Comment (reidab) [reply locations/spiders/campingworld.py]: Updated!

**Pull request merged**